test/common/ssl/ssl_socket_test.cc

```diff
@@ -678,6 +678,159 @@ TEST_P(SslSocketTest, FailedClientAuthSanVerification) {
            GetParam());
 }
 
+TEST_P(SslSocketTest, FailedClientCertificateDefaultExpirationVerification) {
```

Comment:
Please add a sentence or two comment before each test function explaining what it is testing. Many existing tests don't have this, but we're trying to add it for new tests.